bin/server.dart

```diff
@@ -12,10 +11,8 @@ import 'package:badges_bar/badges_bar.dart';
 /// Starts a server that generates SVG badges for pub.dev scores.
 Future<void> main() async {
   await Sentry.init((SentryOptions o) {
-    o.dsn =
-        'https://09a6dc7f166e467793a5d2bc7c7a7df2@o117736.ingest.sentry.io/1857674';
-    o.release = Platform.environment['VERSION'];
-    o.environment = Platform.environment['ENVIRONMENT'];
+    o.dsn = 'https://09a6dc7f166e467793a5d2bc7c7a7df2@o117736.ingest.sentry.io/1857674';
```

Comment:
why not `SENTRY_DSN`?

Reply:
I'd need to set the env var locally when developing (to test Sentry) so just adds more friction.